src/main/java/com/uber/rss/messages/MessageConstants.java

```diff
@@ -57,7 +57,8 @@
     public final static int MESSAGE_ConnectRegistryResponse = -314;
     public final static int MESSAGE_GetBusyStatusRequest = -320;
     public final static int MESSAGE_GetBusyStatusResponse = -321;
-
+    public final static int MESSAGE_ConnectDownloadRequestMessage = -351;
+    public final static int MESSAGE_GetDataAvailabilityRequestMessage = -351;
```

Comment:
maybe let's use a different message code (like `-352`) to avoid the conflict : )

Reply:
done